lib/jsonapi/view.ex

```diff
@@ -130,6 +129,10 @@ defmodule JSONAPI.View do
           do: show(data, conn, params)
       end
 
+      defp host(conn), do: Application.get_env(:jsonapi, :host) || conn.host
```

Comment:
Can you update these to reflect the comment I left?  I made some changes there not in the branch: https://github.com/jeregrine/jsonapi/pull/66#issuecomment-314489047